vendor/milesight-iot/profile-as923.yaml

```diff
@@ -0,0 +1,8 @@
+vendorProfileID: 600
```

Comment:
The idea here is that each profile has its unique index that is issued by the vendor.

The `vendorID` + `vendorProfileID` are globally unique. I see now that you use 600 for different profiles.